docs/toolbox.md

```diff
@@ -0,0 +1,213 @@
+A [toolbox item configuration object](https://surveyjs.io/Documentation/Survey-Creator?id=iquestiontoolboxitem). The [`name`](https://surveyjs.io/Documentation/Survey-Creator?id=iquestiontoolboxitem#name), [`iconName`](https://surveyjs.io/Documentation/Survey-Creator?id=iquestiontoolboxitem#iconName), and [`json`](https://surveyjs.io/Documentation/Survey-Creator?id=iquestiontoolboxitem#json) properties are required. When you configure an object for the `json` property, refer to the API help section of your question type for a list of available properties. For example, if you configure a Dropdown question, refer to the [Dropdown API](https://surveyjs.io/Documentation/Library?id=questiondropdownmodel) help section.
+
+- `index`       
+The target index of the item. Use `0` to add the item at the first position, `1` to add it at the second position, and so on. Do not specify `index` if you want to display the item at the end.
+
+The following code adds a custom item that allows users to select a country from a drop-down menu. Since the `index` argument is `0`, the toolbox displays the item at the beginning.
+
+```js
+creator.toolbox.addItem({
+  name: "countries",
+  iconName: "icon-dropdown",
+  title: "Countries",
+  json: {
+    type: "dropdown",
+    optionsCaption: "Select a country...",
+    choicesByUrl: {
+        url: "https://surveyjs.io/api/CountriesExample"
+    }
+  }
+}, 0);
+```
+
+[View Toolbox Customization example](https://surveyjs.io/Examples/Survey-Creator?id=toolboxcustomization)
+
+<!--
+
+WE HAVEN'T COME UP WITH A VERSION FOR REACT YET
+
+    ### Integrate 3rd-Party Components as Question Editors
+
+    Survey Creator supports integration with the following 3rd-party components out of the box:
+
+    %LIST%
+
+    To enable the integration with one of these components, reference or import the [surveyjs-widgets](https://github.com/surveyjs/custom-widgets) library next to the 3rd-party component sources:
+
+    ```html
+    <script src="https://unpkg.com/surveyjs-widgets/surveyjs-widgets.min.js"></script>
+    ```
+
+    ```js
+    import "survey-creator-core/survey-creator-core.i18n";
+    ```
+
+    If you did not find a desired component in the list above, refer to the following help topic for instructions on how to integrate any 3rd-party component into Survey Creator: [Create Custom Widget](https://surveyjs.io/Documentation/Survey-Creator?id=Create-Custom-Widget).
+-->
+
+<!--  
+
+NEW SURVEY CREATOR DOESN'T HAVE THE ADD TO TOOLBOX ADORNER YET 
+
+    ### Save User-Defined Elements in the Toolbox
+
+    By default, there is a “Add to Toolbox” button on an element (question/panel) in the designer. Your end-user may customize question/panel as he/she wants, add it into toolbox and then drop it on another page.
+
+    You may go even further and persist the current Toolbox state, so the user may use these custom toolbox items for building other surveys.
+
+    Let’s talk here about available options that you have.
```

Comment:
Can we change this line to something like the following?

`The available options are as follows:`